Database/Corrections/Automatic/sodBaseQuests.lua

```diff
@@ -1666,6 +1732,237 @@ function SeasonOfDiscovery:LoadBaseQuests()
+            [questKeys.requiredRaces] = raceIDs.NONE,
+            [questKeys.requiredClasses] = classIDs.NONE,
+            [questKeys.objectivesText] = {"Bring the Brilliant Gold Ring, a Silver Bar, a Moss Agate, and 30 silver coins to Nogg in Orgrimmar."},
+            [questKeys.objectives] = {nil,nil,{{2842},{1206}}},
+        },
+        [80143] = {
+            [questKeys.name] = "Data Rescue",
+            [questKeys.startedBy] = {{7950}},
+            [questKeys.finishedBy] = {{7950,}},
+            [questKeys.requiredLevel] = 40,
+            [questKeys.questLevel] = 40,
+            [questKeys.requiredRaces] = raceIDs.ALL_ALLIANCE,
+            [questKeys.requiredClasses] = classIDs.NONE,
+            [questKeys.objectivesText] = {"Bring a Prismatic Punch Card to Master Mechanic Castpipe in Ironforge."},
+            [questKeys.objectives] = {nil,nil,{{9316}}},
+        },
+        [80153] = {
+            [questKeys.name] = "The Sparklematic 5200!",
+            [questKeys.startedBy] = nil,
+            [questKeys.finishedBy] = nil,
+            [questKeys.requiredLevel] = 40,
+            [questKeys.questLevel] = 40,
+            [questKeys.requiredRaces] = raceIDs.NONE,
+            [questKeys.requiredClasses] = classIDs.NONE,
+            [questKeys.objectivesText] = {"Insert a Grime-Encrusted Item into the Sparklematic 5200, and be sure to have three silver coins to start the machine."},
+            [questKeys.objectives] = {nil,nil,{{9308}}},
+        },
+        [80155] = {
+            [questKeys.name] = "More Sparklematic Action",
+            [questKeys.startedBy] = nil,
+            [questKeys.finishedBy] = nil,
+            [questKeys.requiredLevel] = 40,
+            [questKeys.questLevel] = 40,
+            [questKeys.requiredRaces] = raceIDs.NONE,
+            [questKeys.requiredClasses] = classIDs.NONE,
+            [questKeys.objectivesText] = {"The Sparklematic 5200 is empty, awaiting a grime-encrusted object to clean and three silver coins to start!"},
+            [questKeys.objectives] = {nil,nil,{{9308}}},
+        },
+        [80157] = {
+            [questKeys.name] = "The Sparklematic 5200!",
+            [questKeys.startedBy] = nil,
+            [questKeys.finishedBy] = nil,
+            [questKeys.requiredLevel] = 40,
+            [questKeys.questLevel] = 40,
+            [questKeys.requiredRaces] = raceIDs.NONE,
+            [questKeys.requiredClasses] = classIDs.NONE,
+            [questKeys.objectivesText] = {"As quickly as it started, the Sparklematic 5200 comes to an abrupt halt. &nbsp;A simple "ding" is heard from the machine, and the mouth of the machine spirals upward. &nbsp;Inside the Sparklematic 5200 is a small box wrapped in a little bow. &nbsp;The machine would appear to wrap an object up after eliminating its grime-encrusted status. &nbsp;Even if the machine didn't work, at least the item is now wrapped in a pretty box.Well, there is one way to find out if it cleaned the item or not..."},
```

Comment:
This breaks the code atm.

The `"ding"` ends the string and rips everything apart. Questie doesn't even start ingame and the pipeline also points towards this.

```suggestion
            [questKeys.objectivesText] = {"As quickly as it started, the Sparklematic 5200 comes to an abrupt halt. &nbsp;A simple \"ding\" is heard from the machine, and the mouth of the machine spirals upward. &nbsp;Inside the Sparklematic 5200 is a small box wrapped in a little bow. &nbsp;The machine would appear to wrap an object up after eliminating its grime-encrusted status. &nbsp;Even if the machine didn't work, at least the item is now wrapped in a pretty box.Well, there is one way to find out if it cleaned the item or not..."},
```

We need to cover this case in the crawler though.